metadata_service/proxy/atlas_proxy.py

```diff
@@ -707,7 +707,7 @@ def _get_readers(self, qualified_name: str, top: Optional[int] = 15) -> List[Rea
                     {
                         'attributeName': self.QN_KEY,
                         'operator': 'STARTSWITH',
-                        'attributeValue': qualified_name.split('@')[0]
+                        'attributeValue': qualified_name.split('@')[0] + '.'
```

Comment:
you mean comma or period for seperate?

Reply:
Code is ok, botched the description (should be ok now). Thanks for following up 😅